tests/features/payments.feature

```diff
@@ -77,3 +77,33 @@ Scenario: Delete an existing payment
     Then the server should tell me payment 1 was not found
     When I try to delete a non-existent payment with id 100
     Then I should be returned nothing for payment 100
+
+Scenario: List all payments
+    When I list all "payments"
+    Then I should see "3" existing payments
+    And I should see payment "1" with id "1" and "nickname" = "my credit"
+    And I should see payment "2" with id "2" and "nickname" = "my debit"
+    And I should see payment "3" with id "3" and "nickname" = "my paypal"
+    When I try to delete payment 1
+    When I try to delete payment 2
+    When I try to delete payment 3
```

Comment:
the tedious and not meaningful ways doesn't sounds ideal.

the separate feature file, while it makes sense and is probably the path of least resistence, also seems weird to make a new feature for just one scenario. there are certainly others that could easily move to that no existing data feature, but i kind of like having all of this stuff in one place as they're grouped logically.

finally, a proposal. perhaps you could write a helper method that clears out the DB? call this in your first step when you `get /payments` and assert there's nothing there.

each step should be getting a fresh db to work with so any damage you do in one step does not affect the others.